nsh.8

```diff
@@ -2234,6 +2235,56 @@ nsh(p)/show ospf6
 nsh(p)/
 .Ed
 .Pp
+.Tg pf
+.Ic show pf
+.Op Ar all | anchors | info | labels | memory | queues | rules | sources |
+ states | tables | timeouts | osfingerprint | interfaces
+.Pp
+Display the status of
+.Xr pf 4
+.Ox
+Packet Filter firewall.
+.Nm uses
+.Xr pfctl 8
+with the -s (show) command with modifier arguments to display various aspects
+the current pf status on the system.
+.Bd -literal -offset indent
+nsh(p)/show pf
+% Arguments may be abbreviated
+
+  show pf all           all pf info except fingerprints and interfaces information
+  show pf anchors       currently loaded anchors in main pf ruleset information
+  show pf info          pf filter statistics, counters and tracking information
+  show pf labels        per rule stats (bytes, packets and states) information
+  show pf memory        current pf pool memory hard limit information
+  show pf queues        currently loaded pf queue definition information
+  show pf rules         active pf firewall rule information
+  show pf sources       contents of the pf source tracking table information
+  show pf states        contents of the pf state table information
+  show pf tables        pf table information
+  show pf timeouts      current pf global timeout information
+  show pf osfingerprint pf Operating System fingerprint information
+  show pf interfaces    pf usable interfaces/ interface group information
+nsh(p)/
+.Ed
+.Tg eigrp
```

Comment:
These eigrp documentation additions seem to be unrelated. These were probably meant for a different patch?